molecule/docker/tests/test_localectl.py

```diff
@@ -0,0 +1,19 @@
+# Make sure localectl set's LANG= test value.
+def test_localectl_locale(host):
+    host.run('localectl set-locale LANG=test')
```

Comment:
It looks like your tests are modifying the system, is that right? Wouldn't it be better if lines 3 and 10 were deleted and the assertions only verified that the role properly configures the system?

Reply:
Tests have been modified to only verify the role worked correctly